documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -25,13 +24,13 @@ For example, loadbalancers might not be suitable for certain infrastructure, suc
 
 In this procedure:
 
-. An external listener is configured for the Kafka cluster, with TLS encryption and authentication, and Kafka _simple authorization_ is enabled.
-. A `KafkaUser` is created for the client, with TLS authentication and Access Control Lists (ACLs) defined for _simple authorization_.
+. An external listener is configured for the Kafka cluster, with TLS encryption and authentication, and Kafka _simple authorization_ enabled.
+. A `KafkaUser` is created for the client, with TLS authentication, and Access Control Lists (ACLs) defined for _simple authorization_.
 
-You can configure your listener to use TLS, SCRAM-SHA-512 or OAuth 2.0 authentication.
-TLS always uses encryption, but it is recommended to also use encryption with SCRAM-SHA-512 and OAuth 2.0 authentication.
+You can configure your listener to use TLS, SCRAM-SHA-512, or OAuth 2.0 authentication.
+TLS always uses encryption, but encryption is also recommended when using SCRAM-SHA-512 and OAuth 2.0 authentication.
 
-You can configure simple, OAuth 2.0, OPA or custom authorization for Kafka brokers.
+You can configure simple, OAuth 2.0, OPA, or custom authorization for Kafka brokers.
```

Comment:
Should we capitalize _simple_? Or use it formatter as code as `simple`? I do not think it is really a clear that you do not talk about authorization which is simple but about specific authorization type which is named `simple`. The same seems to be above as well.

```suggestion
You can configure Simple, OAuth 2.0, OPA, or custom authorization for Kafka brokers.
```

Reply:
Updated as follows to make it even clearer:

> You can configure your listener to use `tls`, `scram-sha-512`, or `oauth` authentication.
> TLS always uses encryption, but encryption is also recommended when using SCRAM-SHA-512 and OAuth 2.0 authentication.
> 
> You can configure `simple`, `oauth`, `opa`, or `custom` authorization for Kafka brokers.
> When enabled, authorization is applied to all enabled listeners.